Sources/glTFSceneKit/GLTFConverter+Geometry.swift

```diff
@@ -368,8 +375,17 @@ extension GLTFConverter {
             for primitive in mesh.primitives {
                 // check on draco extension
                 if let dracoMesh = primitive.extensions?[dracoExtensionKey] {
-                    let dracoMesh = dracoMesh as! GLTFKHRDracoMeshCompressionExtension
-                    buffers.insert(glTF.buffer(for: dracoMesh.bufferView))
+                    if let dracoMesh = dracoMesh as? GLTFKHRDracoMeshCompressionExtension {
+                        if let buffer = glTF.buffer(for: dracoMesh.bufferView) {
+                            buffers.insert(buffer)
+                        } else {
+                            errorMessage = GLTFError("Problem with dracoMesh buffer in _preloadBuffersData")
```

Comment:
Change message to: `Can't locate draco buffer with bufferView at \(dracoMesh.bufferView) index`